### PR TITLE
nextpnr: move git clone into earlier build stage

### PR DESCRIPTION
**Pull request by se-bi**
Hey,
as discussed in #4.  

> We can split this into as many PRs as you wish.

Indeed better, to not do it in a big blob.

### CODE REVIEW

File: nextpnr.dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Authors:
 #   Anton Blanchard
 #   Unai Martinez-Corral
+#   Sebastian Birke
 #
 # Copyright 2019-2021 Unai Martinez-Corral <unai.martinezcorral@ehu.eus>
+# Copyright 2021      Sebastian Birke      <git@se-bi.de>
```
Comment (eine) [inline nextpnr.dockerfile:7]: Please, remove this line. It's ok to add the e-mail in line 4 if you wish.

**Review by eine — CHANGES_REQUESTED** [review]
Just the header line. Otherwise, LGTM.

File: nextpnr.dockerfile (hunk 1 above)
Comment (se-bi) [reply nextpnr.dockerfile]: Ok, got it now :see_no_evil: 

just pushed again.

Comment (eine) [reply nextpnr.dockerfile]: No worries! It took me some time to understand too. And each open source project is slightly different in this regard, so you need to "learn" it again rather frequently 😄

**Comment by eine** [comment]
Nice!

**Pull request merged**